utils/apis.ts

```diff
@@ -764,3 +769,85 @@ export async function getTessellationCell(fetcher: $Fetch, tessellationName: str
 
   return maybe.right;
 }
+
+
+// Endpoint: GET /features
+
+export const SceneFeature = t.type({
+  id: t.string,
+  feature_time: t.string,
+  scene: GetSceneResponse,
+});
+
+export type SceneFeatureT = t.TypeOf<typeof SceneFeature>;
+
+export const FeaturesResponse = t.type({
+  features: t.array(SceneFeature),
+});
+
+export async function getFeaturesInRange(fetcher: $Fetch, startTimestamp: number, endTimestamp: number): Promise<SceneFeatureT[]> {
```

Comment:
Would be good to have some brief docs mentioning that the timestamps are indeed Unix timestamps.